rust/cubestore/cubestore/src/lib.rs

```diff
@@ -287,14 +288,9 @@ impl From<flexbuffers::SerializationError> for CubeError {
     }
 }
 
-impl From<s3::S3Error> for CubeError {
-    fn from(v: s3::S3Error) -> Self {
-        let mut m = format!("AWS S3 error: {}", v);
-        if let Some(data) = v.data {
-            m += ": ";
-            m += &data
-        }
-        CubeError::internal(m)
+impl From<anyhow::Error> for CubeError {
+    fn from(v: anyhow::Error) -> Self {
+        CubeError::internal(format!("Error: {}", v.to_string()))
```

Comment:
In new releases `rust-s3` redesigned errors - https://github.com/durch/rust-s3/commit/32a5a69363cb74c86ed8b68f713ca14d8f5a2798#diff-4d994aea29d42345ee9b0840d752f8550afbbb8e56ae458870f030a5213cbf3cR4

it's much better to use v0.32.0 instead of 0.28.1